### ISSUE TITLE
Workspace switching keyboard shortcuts blocked on Solus-Linux

### ISSUE THREAD

**Issue by gnfrazier** [brave/browser-laptop#11377]
### Description
On Solus budgie Linux, Brave blocks native keyboards shortcuts notably ctl+alt+left arrow and ctl+alt+right arrow to change workspaces.   

### Steps to Reproduce

  
Steps to Reproduce:  
Open Brave Browser in workspace 1  
Open Firefox or Chrome in workspace 1  
Make Brave the active application  
use keyboard shortcut ctl+alt+right arrow to go to workspace 2  

**Actual result:**

No action  
**Expected result:**
Switch to workspace 2  
**Reproduces how often:** [What percentage of the time does it reproduce?]
Always
### Brave Version

**about:brave info:**

Brave	0.18.36  
rev	7ab85e9  
Muon	4.3.22  
libchromiumcontent	61.0.3163.79  
V8	6.1.534.32  
Node.js	7.9.0  
Update Channel	dev  
OS Platform	Linux  
OS Release	4.13.5-24.current  
OS Architecture	x64  

**Reproducible on current live release:**

### Additional Information

Switching workspaces does work with Firefox or Chrome Active  
Make Chrome or Firefox the active application
use keyboard shortcut ctl+alt+right arrow to go to workspace 2
Expected:
Switch to workspace 2
Results
Switch to workspace 2  
  
Similar Issues:  
  
Solus may use the Gnome Workspace Grid routines  
Linux: Keyboard Shortcuts Conflict With Gnome 'Workspace Grid' Defaults #7446  
https://github.com/brave/browser-laptop/issues/7446  

OSX Command Keyboard Shortcuts Brave #10085  
https://github.com/brave/browser-laptop/issues/10085

**Comment by Emmanux** [comment]
Most distros depend on these shortcuts. Where can we edit them for Brave?

**Comment by bsclifton** [comment]
@Emmanux unfortunately we currently don't have a way to change them (that I'm aware of). We do have an issue captured for this functionality though, if you wanted to +1 it:
https://github.com/brave/browser-laptop/issues/1374

**Comment by bsclifton** [comment]
Closing as a duplicate of https://github.com/brave/browser-laptop/issues/10085
